### ISSUE TITLE
Not obvious that floating "find" window is scrollable

### ISSUE THREAD

**Issue by cmcaine** [foobnix/LibreraReader#1278]

**Comment by pandadeepimpact** [comment]
Me too. I initially thought that there are only 3 matches, then upon resizing, it was actually 100+ matches.

Though regarding grip lines, it already has, just that instead of dots, they're triangles shaded light blue (if using Light theme). (Tried adding pic, but it doesn't work for some reason.)

**Comment by cmcaine** [comment]
Yes, I saw the triangles, but they don't have lines on so they look less like grip handles to me.
